packages/browser-pool/src/browser-pool.ts

```diff
@@ -471,7 +471,7 @@ export class BrowserPool<
 
         const browserController = await this._launchBrowser(id, { launchOptions, browserPlugin });
         tryCancel();
-        return this._createPageForBrowser(id, browserController, pageOptions);
+        return await this._createPageForBrowser(id, browserController, pageOptions);
```

Comment:
i don't see how this could have any effect, the method is async, this wasn't an ignored promise (and if it was, the issue was on a higher level)